scrutiny.json

```diff
@@ -0,0 +1,82 @@
+{
+    "Scrutiny": {
+        "containers": {
+            "scrutiny": {
+                "image": "ghcr.io/linuxserver/scrutiny",
```

Comment:
I have noticed, there is an official container available. Did you pick the linuxserver one for the ability to just add the "/dev:/dev" variable, so there is no requirement to specify each device manually (which would require the "add volumes" item to be enabled instead)?
@FroggyFlox - in any case, it doesn't look like the official image has multi-arch support like the linuxserver.io does, so this might be the better choice for cross-architecture compatibility until they do (if ever).

Reply:
The official application is based on multiple containers, so for simplicity I chosed to use linuxserver image. 
On my knowledge, the way rockstor is currently managing the devices doesn't really allow to add new devices from the interface, that's why I have chosen to add all of them :smile:

Reply:
ah, ok. I didn't look closely enough to realize that the offical images consists of multiple containers. Thanks for pointing that out.